.travis.yml

```diff
@@ -72,6 +73,9 @@ after_success:
   - git push
   - "[ $TRAVIS_PULL_REQUEST == \"false\" -a $TRAVIS_BRANCH == \"master\" ] && curl -s -X POST -H \"Content-Type:application/json\" -H \"Accept:application/json\" -H \"Travis-API-Version:3\" -H \"Authorization:token $TT\" -d \"{\\\"request\\\":{\\\"branch\\\":\\\"gh-pages\\\", \\\"message\\\":\\\"commit $TRAVIS_COMMIT $TRAVIS_COMMIT_MSG\\\"}}\" https://api.travis-ci.org/repo/mlr-org%2Fmlr-tutorial/requests"
 
+after_script:
+  - Rscript -e 'covr::coveralls()'
```

Comment:
If this fails because of timeout, you could try
```
  - travis_wait 100 Rscript -e 'covr::coveralls()'
```
(if it doesn't fail, just ignore this)